mne/preprocessing/tests/test_maxwell.py

```diff
@@ -128,11 +136,15 @@ def test_maxwell_filter_additional():
     raw_sss.save(test_outname)
     raw_sss_loaded = Raw(test_outname, preload=True, proj=False,
                          allow_maxshield=True)
+
     # Some numerical imprecision since save uses 'single' fmt
     assert_allclose(raw_sss_loaded._data[:, :], raw_sss._data[:, :],
                     rtol=1e-6, atol=1e-20)
 
-    # Test covariance calculation XXX add this
+    # Test covariance calculation
+    cov_raw = compute_raw_data_covariance(raw)
+    cov_sss = compute_raw_data_covariance(raw_sss)
 
+    assert_equal(np.rank(cov_raw), np.rank(cov_sss))
```

Comment:
`cov['nfree']` looks like the number of data points in the covariance calculation (and none of the other keys look like they have rank). I tried: 

`cov_raw_rank = _estimate_rank_meeg_cov(cov_raw['data'], raw.info, scalings)` with `scalings` as `None` or the dict in `test_cov.test_rank()`, but it's just giving me back to total number of channels (or multipolar moments for the SSS analog).